lib/manager/components/transform/ReplaceFileFromVersion.js

```diff
@@ -35,6 +35,21 @@ export default class ReplaceFileFromVersion extends Transform<{}> {
     return issues
   }
 
+  /**
+   * Notify containing component of the resulting validation errors if any.
+   * @param fields: The updated state. If not set, the component state will be used.
+   */
+  _updateErrors = () => {
+    const { onValidationErrors } = this.props
+    if (typeof onValidationErrors === 'function') {
+      onValidationErrors(this.getValidationErrors())
+    }
+  }
+
+  componentDidMount () {
+    this._updateErrors()
+  }
```

Comment:
Generally, I think the pattern we follow is to place lifecycle methods like this at the beginning of the component. Same comment goes for the other files in this PR.